fmriprep/cli/parser.py

```diff
@@ -642,9 +652,18 @@ def parse_args(args=None, namespace=None):
     output_dir = config.execution.output_dir
     work_dir = config.execution.work_dir
     version = config.environment.version
+    output_layout = config.execution.output_layout
 
     if config.execution.fs_subjects_dir is None:
-        config.execution.fs_subjects_dir = output_dir / "freesurfer"
+        if output_layout == "bids":
```

Comment:
good -- you went for `"bids"` (not `"yoda"`), but then why all the examples articulate "YODA" and "yoda/"?   might confuse someone?